transaction/std/src/memo/gift_code_funding.rs

```diff
@@ -136,70 +174,89 @@ mod tests {
 
         // Create memo with blank note
         let note = "";
-        let memo = GiftCodeFundingMemo::new(&key, note).unwrap();
-
-        // Check that the note is extracted properly
-        assert_eq!(memo.funding_note().unwrap(), note);
+        let fee = 666;
+        let memo = GiftCodeFundingMemo::new(&key, fee, note).unwrap();
 
         // Check that the public key can be verified
         assert!(memo.public_key_matches(&key));
+
+        // Check the fee is correct
+        assert_eq!(memo.get_fee(), 666);
+
+        // Check that the note is correct
+        assert_eq!(memo.funding_note().unwrap(), note);
     }
 
     #[test]
-    fn test_gift_code_funding_memo_with_only_null_memo_bytes_is_okay() {
+    fn test_gift_code_funding_memo_note_with_only_null_memo_bytes_is_okay() {
         // Initialize hash bytes
         let mut rng: StdRng = SeedableRng::from_seed([1u8; 32]);
         let key = RistrettoPublic::from_random(&mut rng);
+        let fee: u64 = 666;
```

Comment:
Optional/Nit: Use a suffix instead of declaring a type (throughout this PR)
```suggestion
        let fee = 666u64;
```

Reply:
I think the previous is a bit more readable